### ISSUE TITLE
Rediseño portada wiki

### ISSUE THREAD

**Issue by nukeador** [mozillahispano/mozilla-hispano.org#221]
Hola,

Queremos darle algo de estilo a la portada del wiki ahora que ya no listamos documentación de asistencia.

https://www.mozilla-hispano.org/documentacion/

Las secciones a listar en la portada serían:

* ¿Buscas asistencia? (Esto enlazará a SUMO para despistados)
* ¿Cómo participar en la comunidad? (enlazará al formulario de colaboración)
* Recursos para colaboradores.
* Foros de participación.
* Proyectos.
* Reuniones.
* Eventos.
* Colaboradores.
* Organización de Mozilla Hispano.
* Enlaces a las áreas (sería genial tener un icono actualizado para cada área)

Juan Sebastián ha estado trabajando en algunos bocetos aquí:

http://zemiotic.com/mozilla-iconos/

Abro este issue para hacer el seguimiento de la implementación.

**Comment by nukeador** [comment]
Ejemplo de iconos que usan en mozilla.org

https://www.mozilla.org/en-US/contribute/signup/

**Comment by zemiotic** [comment]
Hola Rubén!

Este fin de semana estuve actualizando el estilo de los iconos para seguir con la línea gráfica de Mozilla. En el enlace se puede ver la nueva versión de los iconos (tienen total afinidad con los del sitio de contribute):

http://zemiotic.com/mozilla-iconos/

¿Cómo nos organizamos para la implementación? ¿la idea es que se muestren en una matriz similar a la página de contribute de Mozilla?

Un saludo!

**Comment by nukeador** [comment]
Me gustan el azul oscuro.

Se pueden hacer pruebas directamente en el wiki creando una página https://www.mozilla-hispano.org/documentacion/Portada/2015 y probando con el css inline.

Aunque lo mejor es integrar el css con un PR en el css de la documentación.

ping @stripTM @fjaguero

**Comment by stripTM** [comment]
Si, por ganar tiempo hay cosas las he hecho apuntando a un css local mío o de un servidor que puedo subir fácilmente.

**Comment by zemiotic** [comment]
Si quieren exporto los iconos de color azul oscuro en un tamaño específico que definamos para que comencemos a hacer pruebas. @stripTM te los envío en png para que hagas pruebas?

**Comment by stripTM** [comment]
@zemiotic envíamelos y también el boceto con el diseño de como quieres que quede la página, lo único aviso que voy a estar una temporada con poco tiempo de seguido, con lo que si hay que tenga más tiempo evitamos que yo sea un cuello de botella.

**Comment by zemiotic** [comment]
Hola @stripTM y @nukeador escribo para retomar esta tarea pendiente de los iconos, en esta semana hice una prueba en HTML y CSS utilizando sprites. Pueden ver el demo en: 

http://zemiotic.com/mozilla-iconos/demo/

También acabo de crear un repositorio de prueba donde está el código fuente.  Les agradezco si me ayudan a mejorarlo y a definir cómo sería la integración en la wiki de Mozilla Hispano.

https://github.com/zemiotic/mozilla-hispano-icons

Quedo atento!

**Comment by nukeador** [comment]
@stripTM ¿Tienes tiempo para coordinate con @zemiotic y ver de implementarlo?

**Comment by nukeador** [comment]
Hola,

¿En qué quedó esto?

un saludo.

**Comment by sebastianserna** [comment]
Hola!

Está frenado, el último avance fue la prueba HTML con imágenes tipo sprite en PNG que hice en este repositorio:

https://github.com/zemiotic/mozilla-hispano-icons

También podríamos hacerlas con gráficos SVG. En la carpeta vector del repositorio están los archivos fuente vectoriales de los iconos. ¿Cómo podríamos integrarlos en la wiki?

Un saludo!

**Comment by nukeador** [comment]
@fjaguero @stripTM ¿podéis ayudar a @sebastianserna a integrar esto?

**Comment by fjaguero** [comment]
@nukeador @sebastianserna Claro. ¿La idea es integrar lo que está en la URL de la demo? ¿Serían esas 8 o habría que añadir algo más?

**Comment by nukeador** [comment]
Sería lo que está, que es lo mismo que listamos ahora en la portada del wiki.

**Comment by fjaguero** [comment]
Ok, yo puedo encargarme. @stripTM @nukeador ¿La idea es meter el CSS de esa sección en este file? https://github.com/mozillahispano/mozilla-hispano.org/blob/master/documentacion/skins/bookjive/main.css

¿Qué textos se mantienen de la wiki actual? ¿Sería usar ese boceto exacto (quitando por ej. el módulo de reuniones y dejando solo el icono, etc)?

**Comment by nukeador** [comment]
El theme ese es el viejo. @stripTM hizo uno nuevo para la versión nueva del wiki, pero parece que no está en el repo? :?

**Comment by stripTM** [comment]
Está en la rama prod https://github.com/mozillahispano/mozilla-hispano.org/tree/prod/documentacion/skins

¿podríamos quedar un día para estabilizar las ramas master y prod?
